src/pal/src/exception/signal.cpp

```diff
@@ -199,6 +199,7 @@ Return :
 void FreeSignalAlternateStack()
 {
     stack_t ss, oss;
+    ss.ss_size = MINSIGSTKSZ;
```

Comment:
Maybe add a comment that this field is not actually ignored for `SS_DISABLE` on MUSL?

Reply:
That's weird - I have added that comment, I must have forgotten to push that change.